README.md

```diff
@@ -1,21 +1,21 @@
-## Sewer          
+## Sewer
 
```

Comment:
@AlecTroemel generally in the `README.md` file, spaces are important at the end of lines so that markdown can create new lines.   
I would like to keep the markdown formatting as is on the master branch, if you do disagree with that formatting; I would be happy to discuss such disagreement in a new github issue separate from this PR

Reply:
@komuw I have no problem with the master branches formatting. I have a setting in my text editor that removes trailing whitespace, and theses changes just slipped through. I'll undo them!